### ISSUE TITLE
The Accuracy

### ISSUE THREAD

**Issue by LZH101** [LanglandsLin/ActCLR#13]
Hi, I have a question. I am using a single GPU, base_lr is 0.1, and pytorch is 1.8.1. But the joint accuracy in xview of NTU60 is 81.6. There is a 5% gap (huge) from the published results. What is the problem?

**Comment by LanglandsLin** [comment]
Feel free to come and share ideas! @LZH101 
I've updated the config file for the single card training, and the weights and accuracies obtained from the corresponding training. We do notice a different accuracy with the 4-card one. The reduced batchsize due to the single card leads to that the pre-training may be more prone to overfitting. Our accuracy at 350 epoch is 85.81, while if we choose 300 epoch weights for linear evaluation, the accuracy is 86.45, both with some decrease from multi-card training. These updated weights and results have been updated for reference. Perhaps adding some regularization or gradient accumulation could mitigate the drop in accuracy due to overfitting, but this part has not been validated.
